<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1500</width>
    <height>800</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <kerning>true</kerning>
   </font>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(255, 255, 255);</string>
  </property>
  <property name="toolButtonStyle">
   <enum>Qt::ToolButtonTextUnderIcon</enum>
  </property>
  <property name="tabShape">
   <enum>QTabWidget::Rounded</enum>
  </property>
  <widget class="QWidget" name="centralwidget"/>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1500</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="toolBar">
   <property name="sizePolicy">
    <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(51,51,51);
color: rgb(189, 191, 192);
font-weight: bold;
font-size: 10px;
spacing: 5px;</string>
   </property>
   <property name="movable">
    <bool>false</bool>
   </property>
   <property name="orientation">
    <enum>Qt::Vertical</enum>
   </property>
   <property name="iconSize">
    <size>
     <width>64</width>
     <height>40</height>
    </size>
   </property>
   <property name="toolButtonStyle">
    <enum>Qt::ToolButtonTextUnderIcon</enum>
   </property>
   <attribute name="toolBarArea">
    <enum>LeftToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionHome"/>
   <addaction name="actionTrade"/>
   <addaction name="actionPerformance"/>
   <addaction name="actionNews"/>
   <addaction name="actionMarkets"/>
   <addaction name="separator"/>
  </widget>
  <action name="actionHome">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="../resources/resources.qrc">
     <normaloff>:/home.png</normaloff>
     <activeon>:/home_on.png</activeon>:/home.png</iconset>
   </property>
   <property name="text">
    <string>Home</string>
   </property>
  </action>
  <action name="actionTrade">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="../resources/resources.qrc">
     <normaloff>:/trade.png</normaloff>
     <activeon>:/trade_on.png</activeon>:/trade.png</iconset>
   </property>
   <property name="text">
    <string>Trade</string>
   </property>
  </action>
  <action name="actionPerformance">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="../resources/resources.qrc">
     <normaloff>:/performance.png</normaloff>
     <activeon>:/performance_on.png</activeon>:/performance.png</iconset>
   </property>
   <property name="text">
    <string>Performance</string>
   </property>
  </action>
  <action name="actionNews">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="../resources/resources.qrc">
     <normaloff>:/news.png</normaloff>
     <activeon>:/news_on.png</activeon>:/news.png</iconset>
   </property>
   <property name="text">
    <string>News</string>
   </property>
  </action>
  <action name="actionMarkets">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="../resources/resources.qrc">
     <normaloff>:/markets.png</normaloff>
     <activeon>:/markets_on.png</activeon>:/markets.png</iconset>
   </property>
   <property name="text">
    <string>Markets</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="../resources/resources.qrc"/>
 </resources>
 <connections/>
</ui>
